SparkPost/2web2ui
# FE-927: Updated billing components

### What Changed
Updated:
- Billing address form
- Payment Form
 - Plan price (on plan select)

### How To Test
 - Go to /account/billing
 - Verify that the update billing info and payment information match styles
 - Go to /account/billing/plan
 - Verify that the styles are matching correctly

### To Do
- [ ] Update the global banner
- [ ] Possibly update card summary? Not sure if anything needs to be done here
